client/landing/jetpack-cloud/sections/scan/main.jsx

```diff
@@ -3,22 +3,103 @@
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { withLocalizedMoment } from 'components/localized-moment';
+import SecurityIcon from 'landing/jetpack-cloud/components/security-icon';
+
+import './style.scss';
 
 class ScanPage extends Component {
+	renderScanOkay() {
+		const { siteSlug, moment, lastScanTimestamp, nextScanTimestamp } = this.props;
+
+		let nextScanString = translate( 'tomorrow' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **126** times:
`translate( 'Tomorrow' )` *ES Score: 13*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/scan-okay)